executor/simple.go

```diff
@@ -377,19 +377,16 @@ func (e *SimpleExec) executeFlush(s *ast.FlushStmt) error {
 		defer sysSessionPool.Put(ctx)
 		err = dom.PrivilegeHandle().Update(ctx.(sessionctx.Context))
 		return errors.Trace(err)
-	case ast.FlushStatus:
-		dom := domain.GetDomain(e.ctx)
-		if plugin.Get(plugin.Audit, "ipwhitelist") != nil {
-			if cli := dom.GetEtcdClient(); cli != nil {
-				const whitelistKey = "/tidb/plugins/whitelist"
-				row := cli.KV
-				_, err := row.Put(context.Background(), whitelistKey, "")
-				if err != nil {
-					log.Warn("notify update whitelist failed:", err)
-				}
+	case ast.FlushTiDBPlugin:
```

Comment:
`FlushTiDBPlugin` is irrelevant to this PR, you can do it in a separate one.

Reply:
ok